react/week4/searchegit/src/App.js

```diff
@@ -0,0 +1,37 @@
+import React, {useState} from 'react';
+import './App.css';
+import SearcheGit from './SearcheGit'
```

Comment:
The common approach is to make a components folder and put js files for components there ie
import SearchGit from '../components/SearchGit';
```suggestion
import SearchGithub from './SearchGithub'
```